frontend/src/pages/SignIn.tsx

```diff
@@ -137,7 +137,7 @@ export class SignIn extends React.Component<Props, State> {
             if (next != null && next !== "") {
                 return <Redirect  to={next} />;
             }
-            return <Redirect  to="/units/" />;
+            return <Redirect  to="/devices/" />;
```

Comment:
It looks like there are extra spaces between the `Redirect` and the `to`. Please kill them off in a future PR.